lib/channel/Player.ts

```diff
@@ -90,30 +91,44 @@ export class Player extends Base {
     return this.__volume;
   }
 
+  public getPlayList() : Playlist {
```

Comment:
@konrad-marzec what is the purpose of making this getter? I think we should rather expose playlist just at the playlist-fetched callback? Isn't that enough? I would prefer to avoid getters that refer to "global state".